.circleci/config.yml

```diff
@@ -75,14 +75,27 @@ commands:
           destination: test_results
 
 jobs:
-  unit_test:
+  linting_checks:
     <<: *job_defaults
     steps:
       - test_setup
       - run:
-          name: Check Python lint
+          name: Check Python lint with yapf
           working_directory: ~/curation
           command: tools/yapf_lint.sh
+      - run:
+          name: Pylint on data_steward
+          working _directory: ~/curation
+          command: pylint -E data_steward/
+      - run:
+          name: Pylint on tests
+          working_directory: ~/curation
+          command: pylint -E tests/
```

Comment:
Could this be combined with the previous command e.g. like this?
`pylint -E data_steward tests`

Reply:
we can run but I thought it's better if we have a separation.